js/webp-fix.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * webp-fix.js
+ *
+ *	Detects lack of webp support, loads full images instead. 
+ *
+ * Usage:
+ *   $config['additional_javascript'][] = 'js/jquery.min.js';
+ *   $config['additional_javascript'][] = 'js/webp-fix.js';
+ *
+ */
+
+$(document).ready(function() {
+
+	// from: https://stackoverflow.com/a/54120785
+	function testWebP() {
+		return new Promise(res => {
+			const webP = new Image();
+			webP.src = 'data:image/webp;base64,UklGRjoAAABXRUJQVlA4IC4AAACyAgCdASoCAAIALmk0mk0iIiIiIgBoSygABc6WWgAA/veff/0PP8bA//LwYAAA';
+			webP.onload = webP.onerror = () => {
+				res(webP.height === 2);
+			};        
+		})
+	};
+
+	testWebP().then(hasWebP => {
+		if(!hasWebP){
+			$('div.file a').each(function(index){
+				let ext = ["png","jpg", "jpeg"];
```

Comment:
This is constant, should it be moved up one line to be outside of the loop? This would avoid it being defined for each image (possibly hundreds).